TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts

```diff
@@ -102,3 +103,11 @@ tasks.register<Copy>("copyKeysJSON") {
     into(layout.projectDirectory.dir("dafny/TestVectorsAwsCryptographicMaterialProviders/test"))
 }
 
+tasks.register<Copy>("copyKeysJSONCurr") {
+    from(layout.projectDirectory.file("../../dafny/TestVectorsAwsCryptographicMaterialProviders/test/keys.json"))
+    into(layout.projectDirectory.dir("."))
+}
```

Comment:
Do we really want this as a task?
I'm just wondering what the level of indirection gives us?

Reply:
we technically don't need it, I was just getting confused with the `cp` command and found it easier to just call this make task...